### PR TITLE
Update calendar.py

### PR DESCRIPTION
**Pull request by Ombental**
The get method of attachment updating didn't have a relevant endpoint.

I don't know how to handle the attachments changing except for the built in method in the attachments class...

### CODE REVIEW

File: O365/calendar.py [hunk 1]
```diff
@@ -958,9 +961,13 @@ def to_api_data(self, restrict_keys=None):
         if self.__recurrence:
             data[cc('recurrence')] = self.__recurrence.to_api_data()
 
-        if self.has_attachments:
-            data[cc('attachments')] = self.__attachments.to_api_data()
-
+#         if self.has_attachments:
+#             data[cc('attachments')] = self.__attachments.to_api_data()
+#  Event object doesn't have __attachments attribute...
```
Comment (Ombental) [inline O365/calendar.py:966]: useless comment

Comment (janscas) [inline O365/calendar.py]: `Event` object does have the "__attachments" object. So this should be included (not commented out)

File: O365/calendar.py [hunk 2]
```diff
@@ -958,9 +961,13 @@ def to_api_data(self, restrict_keys=None):
...
+        
+        if 'attachments' in restrict_keys:
+            self.attachments._update_attachments_to_cloud()
```
Comment (janscas) [inline O365/calendar.py:965]: That's ok

**Review by janscas — COMMENTED** [review]
I can't merge this as it will not include the attachments if there's an update because you commented a key piece of code. Can u revert it?

The other changes are ok and should fix the get attachment method.

**Comment by janscas** [comment]
thanks!

**Pull request merged**